coalib/parsing/LineParser.py

```diff
@@ -56,6 +58,17 @@ def parse(self, line):
             line,
             self.comment_separators)
         comment = unescape(comment)
+
+        if line.startswith('#'):
+            if(line[1] != ' '):
+                logging.warning('Your line has no whitespace after #')
+
+        if (line != '' and line.find('#') != -1):
+            pos = [i for i, x in enumerate(line) if x == '#']
```

Comment:
Shouldn't we get only the first one? This would emit a warning for something like

```
files = **.py  # check everything! (#yolo)
```

Not the smartest example but should illustrate my point :D